From eddf57fb9748791e021ef550d651cc72c48add5c Mon Sep 17 00:00:00 2001
From: Koen Kooi <koen@openembedded.org>
Date: Thu, 15 May 2008 09:32:23 +0200
Subject: [PATCH] ARM: OMAP: board-omap3beagle: fix merge-damage in RTC code

This patch fixes the merge-damage in the beagleboard RTC code

Signed-off-by: Koen Kooi <koen@openembedded.org>
---
 arch/arm/mach-omap2/board-omap3beagle.c |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-omap2/board-omap3beagle.c b/arch/arm/mach-omap2/board-omap3beagle.c
index 0c15ca0..0c0cbfc 100644
--- a/arch/arm/mach-omap2/board-omap3beagle.c
+++ b/arch/arm/mach-omap2/board-omap3beagle.c
@@ -76,6 +76,7 @@ static struct platform_device *omap3_beagle_devices[] __initdata = {
 
 static void __init omap3_beagle_init(void)
 {
+	platform_add_devices(omap3_beagle_devices, ARRAY_SIZE(omap3_beagle_devices));
 	omap_board_config = omap3_beagle_config;
 	omap_board_config_size = ARRAY_SIZE(omap3_beagle_config);
 	omap_serial_init();
@@ -88,8 +89,6 @@ arch_initcall(omap3_beagle_i2c_init);
 
 static void __init omap3_beagle_map_io(void)
 {
-	platform_add_devices(omap3_beagle_devices,
-				ARRAY_SIZE(omap3_beagle_devices));
 	omap2_set_globals_343x();
 	omap2_map_common_io();
 }
-- 
1.5.4.3

